### PR TITLE
Home refactoring & tests

### PR DESCRIPTION
**Pull request by DmytroDidukh**
- refactoring according to readme
- tests to theme redux
- cypress tests to homepage and cabinet

### CODE REVIEW

File: src/services/getItems.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
```
Comment (VolodymyrTrach) [inline src/services/getItems.js:1]: This service is outdated. Use apollo client getItems()

**Comment by VolodymyrTrach** [comment]
pull development again

File: src/pages/home/categories/categories.js [hunk 2]
```diff
@@ -2,24 +2,27 @@ import React from 'react';
...
 
-import LoadingBar from '../../../components/LoadingBar';
-import CategoryItem from '../category-item';
+import LoadingBar from '../../../components/loading-bar';
+import CategoryItem from './category-item';
 import { useStyles } from './categories.style';
-import { LANGUAGE, HOMEPAGE_TITLES } from '../../../configs';
+import { HOMEPAGE_TITLES } from '../../../configs';
 
 const Categories = () => {
```
Comment (Alan18081) [inline src/pages/home/categories/categories.js:10]: Please, rename component and its files to `CategoriesList` to reduce duplication of code with a name of the state key

File: src/redux/categories/tests/categories.reducer.test.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import categoriesReducer from '../categories.reducer';
 import { setCategories, setCategoriesLoading } from '../categories.actions';
 
-describe('Categories reducer test', () => {
+describe('CategoriesList reducer test', () => {
```
Comment (Alan18081) [inline src/redux/categories/tests/categories.reducer.test.js:4]: It should still be called `Categories`

File: src/redux/categories/tests/categories.sagas.test.js [hunk 4]
```diff
@@ -6,7 +6,7 @@ import getItems from '../../../utils/client';
 import { SET_CATEGORIES, SET_CATEGORIES_LOADING } from '../categories.types';
 import { SET_ERROR } from '../../error/error.types';
 
-describe('Categories saga', () => {
+describe('CategoriesList saga', () => {
```
Comment (Alan18081) [inline src/redux/categories/tests/categories.sagas.test.js:9]: It should still be called `Categories`

File: src/redux/categories/tests/categories.actions.test.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
 import { setCategories, getCategories } from '../categories.actions';
 import { GET_CATEGORIES, SET_CATEGORIES } from '../categories.types';
 
-describe('Categories actions test', () => {
+describe('CategoriesList actions test', () => {
```
Comment (Alan18081) [inline src/redux/categories/tests/categories.actions.test.js:4]: It should still be called `Categories`

File: src/pages/home/categories/categories-list.js [hunk 6]
```diff
@@ -42,4 +45,15 @@ const Categories = () => {
   );
 };
 
-export default Categories;
+export const getCategoryURL = (category) => {
+  console.log(category, '2');
```
Comment (Alan18081) [inline src/pages/home/categories/categories-list.js:49]: Remove useless `console.log`

**Pull request merged**